src/Components/Shifting/ListFilter.tsx

```diff
@@ -117,10 +118,11 @@ export default function ListFilter(props: any) {
 
   useEffect(() => {
     async function fetchData() {
+      console.log("fetching assigned user");
       if (filter.assigned_user) {
         setAssignedUserLoading(true);
         const params = {
-          limit: 10,
+          limit: PAGE_LIMIT,
```

Comment:
According to PR title, only 14 should be replaced by `PAGE_LIMIT`.  😂 

Hmm there were so inconsistency in page limits.